### DISCUSSION TITLE
Добавить HwV380E21_WF3_IPC38_ETH_2016_0217

### DISCUSSION THREAD

**Discussion by VReunov** [Arkady23/V380-Web#27] [Q&A]
Пробовал httpd_V380E3_C3_3020607_v20210620 - не подошло. папка cgi-bin не появилась. Прошу добавить данную модель
![111](https://user-images.githubusercontent.com/2165575/150636552-55968b35-5cf4-4223-842e-f383eae65cf3.png)

**Comment by Arkady23** [comment]
Мне нужно знать точный номер модели и версию. А этот патч пробовали https://github.com/Arkady23/V380-Web/releases/download/20210530/httpd_V380E3_C3_v20210606.zip ?
Если приобретали на алиэкспресс — дайте ссылку. Посмотреть, что за камера.
Если всё безуспешно, тогда надо запрашивать прошивку у производителя, как это делали некоторые другие https://github.com/Arkady23/V380-Web/discussions/20 Т.к. вытащить модель и версию можно из любого подходящего именно к вашей камере патча.

**Reply by VReunov** [reply]
Камеру с [озона](https://www.ozon.ru/products/255996343/) брал. Прошивку выпросил. После установки еще раз попробую поставить что есть тут. Или не надо обновлять?
[AK_3918E-V300__p2_V3.3.2.9_universal_upgrade_patch_2020-11-30.zip](https://github.com/Arkady23/V380-Web/files/7925819/AK_3918E-V300__p2_V3.3.2.9_universal_upgrade_patch_2020-11-30.zip)

**Reply by Arkady23** [reply]
У вас в патче в основном файле указана модель V380E3_C5. Такой патч в архиве есть https://github.com/Arkady23/V380-Web/releases/download/20210530/httpd_V380E3_C5_v20210607.zip
Если установится, то обновите сервер средствами web-сервера на последнюю версию https://github.com/Arkady23/V380-Web/releases/download/20210620/update_20210620.zip

**Reply by Arkady23** [reply]
Не знаю, нужно ли обновлять или нет. Лучше не надо. Если всё будет работать, как вам надо — не обновляйте этим патчем. В этом патче mp3 файлы на китайском (не на английском). Если будет официальное обновление, то приложение на смартфоне вам должно подсказать, что вышло обновление. Чтобы узнать есть ли обновление, зайдите в приложение в пункт о версии прошивке, этот пункт как-то так называется.

**Reply by Arkady23** [reply]
Добавил в таблицу.... И ссылку на Озон тоже.

**Reply by VReunov** [reply]
Не работает. Папка веб-морды не появилась

**Reply by Arkady23** [reply]
IP правильно набираете?

**Reply by Arkady23** [reply]
А было на английском, что идет обновление? Папка cgi-bin не появилась?

**Reply by VReunov** [reply]
да, 80 порт не отвечает, телнет тоже не подключает

**Reply by VReunov** [reply]
![Снимок экрана 2022-01-24 в 17 14 12](https://user-images.githubusercontent.com/2165575/150798947-4716ad6e-29f2-49b3-825c-027eb3936809.png)
Камера на улице, высоко стоит. После ее включения не слышно ее.

**Reply by Arkady23** [reply]
В патче, который вы опубликовали указан номер версии патча 3030209. Если вы его установили, то я могу вам сделать патч под номером 3030210. Если не установили, то под номером 3030208.

**Reply by VReunov** [reply]
Открыты только 554, 8800 и 8899 порты

**Reply by VReunov** [reply]
Не устанавливал

**Reply by Arkady23** [reply]
Сделал такую версию https://github.com/Arkady23/V380-Web/releases/download/20210530/httpd_V380E3_C5_3030210_v20210620.zip

**Reply by VReunov** [reply]
Не завелось. Папка cgi-bin не появилась

**Reply by Arkady23** [reply]
Возможно на поддержке вам дали патч не от вашей камеры. Может такое быть?

**Reply by Arkady23** [reply]
Попробуйте трафик выгрузить.

**Reply by Arkady23** [reply]
Может попробовать сделать патч для модели V380E3_C4?

**Reply by VReunov** [reply]
Да всякое может быть. Пофиг, не заработало и ладно. Есть вариант хотя бы telnet запустить?
Кстати, я не нашёл строку base64 выгружая трафик. Перепробовал несколько версий приложений. Есть только json ответ информации обустройстве.
`{"id":xxxx,"method":"login","params":{"version":31,"phoneType":1012,..`

`	"id":	xxx,
	"result":	{
		"code":	1001,
		"errorId":	148
	},
	"v380":	{
		"version":	3,
		"dId":	xxx,`

**Reply by Arkady23** [reply]
В том то и дело, что первое, что надо это узнать модель камеры. Вроде я описал в описании как это можно сделать и обучающее видео снял.

**Reply by VReunov** [reply]
Ну нет такой строки. Вон у товарища тоже ее не было. https://github.com/Arkady23/V380-Web/discussions/9#discussioncomment-940234
Они может спрятали ее, хз.

**Reply by Arkady23** [reply]
У этого человека камера, как у других двух и у всех проблемы с доступом к ней. Я уже таких камер заказал 2 шт. у разных продавцов. Жду когда придут, посмотрю.

**Reply by Arkady23** [reply]
Про вашу камеру ничего сказать не могу. Пришлите трафик, чтобы я сравнил со своим. Очень часто люди снимают не тот трафик.

**Reply by Arkady23** [reply]
По тому человеку, на которого вы сослались, вернее по таким камерам, как у него могу сказать, что трафик у них шифрованный трафик https, а на моих камерах нешифрованный http. Это меня насторожило, возможно китайцы изменили методику обновлений и теперь и модель камеры не узнать и патч не получить. Придёт камера — попробую поразбираться.

**Reply by VReunov** [reply]
Вот кст да. Могу в вк отправить дамп

**Reply by Arkady23** [reply]
Обновил все патчи. Пробуйте ваш https://github.com/Arkady23/V380-Web/releases/download/20210530/httpd_V380E3_C5_9090909_v20210620.zip
Должно встать!
